### DISCUSSION TITLE
Tractor-leveller combination for silage compacting

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/Courseplay_FS22#2780] [Q&A]
Hey,

What combination of tractor and silage leveller(shield) are you guys using?

It seems like no matter what combo I pick, the tractor never manages to climb the heap, instead it shoves the blade into the ground at the foot of the silo.

Do you guys know what vehicle combination I can use?

**Comment by Tensuko** [comment]
All 3 Vanilla shields and they work fine for me.
We just disabled the tilting as ppl said they create the bumps.

**Comment by Tensuko** [comment]
Example of the result and the used blade.
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/d69843e7-4d2a-4902-b060-8183a85c9d10)

**Comment by nitroSkirata** [comment]
ok, I tried it a bit more, the siloworker does indeed mange to sometimes climb the silo to compact it, but most of the time it is unable to move up the heap.

Used a 1050 with the 5.4m blade.

the problem is not that it isn't pushing the silage, it is that the silage isn't compacted, as the worker only pushes the front of the silo together instead of driving over the top of the pushed together pile.

**Reply by Tensuko** [reply]
A shield is not for compacting....
